Extensions/PathfindingBehavior/Extension.cpp

```diff
@@ -559,13 +575,23 @@ void DeclarePathfindingBehaviorExtension(gd::PlatformExtension& extension) {
         .SetFunctionName("GetCellHeight")
         .SetIncludeFile("PathfindingBehavior/PathfindingRuntimeBehavior.h");
 
+    aut.AddExpression("MovementAngle",
+                    _("Angle of movement"),
+                    _("Angle of displacement of an object"),
+                    _("Path"),
+                    "CppPlatform/Extensions/AStaricon16.png")
+        .AddParameter("object", _("Object"))
+        .AddParameter("behavior", _("Behavior"), "PathfindingBehavior")
+        .SetFunctionName("GetMovementAngle")
```

Comment:
You can remove this FunctionName as it's not implemented in the native engine. Also remove the SetIncludeFile